### PR TITLE
docs(replays): add API documentation for replay-selectors endpoint

### PR DESCRIPTION
**Pull request by michellewzhang**
Related: https://github.com/getsentry/team-replay/issues/353

### CODE REVIEW

File: src/sentry/apidocs/examples/replay_examples.py [hunk 1]
```diff
@@ -58,3 +58,30 @@ class ReplayExamples:
             response_only=True,
         ),
     ]
+
+    GET_SELECTORS = [
+        OpenApiExample(
+            "Retrieve a collection of selectors for an organization.",
+            description="Returns a list of objects with selector attributes.",
```
Comment (schew2381) [inline src/sentry/apidocs/examples/replay_examples.py:65]: I think you can exclude this field, but it's also ok if you want to leave it

**Review by schew2381 — APPROVED** [review]

File: src/sentry/replays/endpoints/organization_replay_selector_index.py [hunk 2]
```diff
@@ -59,7 +63,13 @@ def get_replay_filter_params(self, request, organization):
         return filter_params
 
     @handled_snuba_exceptions
+    @extend_schema(
+        operation_id="List an Organization's Selectors",
+        parameters=[GlobalParams.ORG_SLUG, ReplaySelectorValidator],
+        examples=ReplayExamples.GET_SELECTORS,
```
Comment (schew2381) [inline src/sentry/replays/endpoints/organization_replay_selector_index.py:69]: I think this is missing the cursorqueryparam, and also the response body. Looks like there's no typeddict response body too 😢

Comment (JoshFerge) [reply src/sentry/replays/endpoints/organization_replay_selector_index.py]: can we ensure that the docs test fails if there's no response body schema?

Comment (schew2381) [reply src/sentry/replays/endpoints/organization_replay_selector_index.py]: It was failing in the openapi-diff / build api action was failing at the [build](https://github.com/getsentry/sentry/actions/runs/7806938986/job/21294303397?pr=64713).
@sentaur-athena If the build itself fails (not the diff), is that still non-blocking?

Comment (JoshFerge) [reply src/sentry/replays/endpoints/organization_replay_selector_index.py]: yup if the build fails that is blocking!

**Review by JoshFerge — APPROVED** [review]

**Review by cmanallen — CHANGES_REQUESTED** [review]

File: src/sentry/replays/endpoints/organization_replay_selector_index.py [hunk 3]
```diff
@@ -59,7 +87,23 @@ def get_replay_filter_params(self, request, organization):
...
 
     @handled_snuba_exceptions
+    @extend_schema(
+        operation_id="List an Organization's Selectors",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+            ReplaySelectorValidator,
+            CursorQueryParam,
+            VisibilityParams.PER_PAGE,
+        ],
```
[the remark this answers is not in the document]
Comment (michellewzhang) [reply src/sentry/replays/endpoints/organization_replay_selector_index.py:97]: aw sad, they were missing from the blueprint :(

**Review by cmanallen — APPROVED** [review]

**Pull request merged**